<head>
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
  <link rel="stylesheet" href="https://hive.biochemistry.gwu.edu/prd//gcsslib/bootstrap.min.css" type="text/css">
  <link rel="stylesheet" href="https://hive.biochemistry.gwu.edu/prd//gcsslib/global.css" type="text/css">
  <link rel="stylesheet" href="https://hive.biochemistry.gwu.edu/prd//gcsslib/googlefonts.css" type="text/css">


    <!-- Google Tag Manager -->
    <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
    new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
    j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
    'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
    })(window,document,'script','dataLayer','GTM-TQNKGGW');</script>
    <!-- End Google Tag Manager -->

</head>

<body>
<div vocab="http://schema.org/" typeof="WebPage">
<table style="width:100%;font-size:13px;">
<tr>
<td style="width:100%; vertical-align: top;" colspan="2">
<h3 id="Top">HIVE Team Datasets:</h3>
</td>
</tr>
<tr style="height: 78px;">
<td style="width: 40px; text-align: right; vertical-align: top; height: 78px;">
<p>&nbsp;</p>
</td>
<td style="vertical-align: top; width:100%; height: 78px;font-size:13px;">
<ol>

<li>
<div vocab = "http://schema.org/" typeof="Dataset">
  <span property="name">BCO HCV</span><br>
  <span property="abstract">
        We demonstrated that the use of the IEEE 2791-2020 Standard, (BioCompute objects [BCO]) enables complete and concise communication of NGS data analysis results. One arm of a clinical trial4 was replicated using synthetically generated data made to resemble real biological data. Two separate, independent analyses were then carried out using BCOs as the tool for communication of analysis: one to simulate a pharmaceutical regulatory submission to the FDA, and another to simulate the FDA review. The two results were compared and tabulated for concordance analysis: of the 118 simulated patient samples generated, the final results of 117 (99.15%) were in agreement. This high concordance rate demonstrates the ability of a BCO, when a verification kit is included, to effectively capture and clearly communicate NGS analyses within regulatory submissions. BCO promotes transparency and induces reproducibility, thereby reinforcing trust in the regulatory submission process.
  </span>
  
  <br>
  
  <br><a href="HTMLROOT/content/soft_links/P0641M00001_S1_L001_R1_001.fastq" download>Base line 1</a>
  Version:
  <span property="version">1.0 </span>
  Size:
  <span property="size">48 MB</span>
  Format:
  <span property="encodingFormat">FASTQ</span>

  <br><a href="HTMLROOT/content/soft_links/P0641M00001_S1_L001_R2_001.fastq" download>Base line 2</a>
  Version:
  <span property="version">1.0 </span>
  Size:
  <span property="size">48 MB</span>
  Format:
  <span property="encodingFormat">FASTQ</span>
  <br>

  <br><a href="HTMLROOT/content/soft_links/P0641M00002_S2_L001_R1_001.fastq" download>Treatment Failure 1</a>
  Version:
  <span property="version">1.0 </span>
  Size:
  <span property="size">48 MB</span>
  Format:
  <span property="encodingFormat">FASTQ</span>

  <br><a href="HTMLROOT/content/soft_links/P0641M00002_S2_L001_R2_001.fastq" download>Treatment Failure 2</a>
  Version:
  <span property="version">1.0 </span>
  Size:
  <span property="size">48 MB</span>
  Format:
  <span property="encodingFormat">FASTQ</span>
  <br>

  <br><b>Full dataset download</b><br>
  These files are output from the HIVE platform. The zip file has a subfile for each read, named with a number. The number is the read id from the HIVE platform (ex: 580561). Inside each of those files are a few other metadata CSV files and the actual sequence is ANOTHER zip file titled "_.zip". Once you uncompress that it should be named something like "P0641M00002_S2_L001_R2_001.fastq". This is sample #2, Reverse (R2) read. "P0641M00002_S2_L001_R1_001.fastq" would be sample #2 Forward (R1) read. Each object in the manifest.json file mapps the HIVE ID  to the read sample. 
<br>
  <br><a href="HTMLROOT/content/soft_links/manifest.json" download>manifest.json</a>
  Version:
  <span property="version">1.0 </span>
  Size:
  <span property="size">163K </span>
  Format:
  <span property="encodingFormat">JSON</span>

  <br><a href="HTMLROOT/content/soft_links/hcvALL.zip" download>hcvALL.zip</a>
  Version:
  <span property="version">1.0 </span>
  Size:
  <span property="size">4.4G </span>
  Format:
  <span property="encodingFormat">ZIP</span>
  <br>

  <br>Citation:
  <span property="citation"><a href="https://doi.org/10.1101/2020.12.07.415059">doi: https://doi.org/10.1101/2020.12.07.415059</span>


</div>
</li>
<br>
<li>
  <span property="name">GFKB</span><br>
  <span property="abstract">
        Gut feeling knowledgebase is a reference database of healthy human gut microbiome. It is generated by a metagenomic analysis pipeline describes in our paper (doi: 10.1371/journal.pone.0206484), and includes three tools which are integrated in the HIVE platform. 49 healthy samples sequenced at GWU and 49 healthy samples taken from The Human Microbiome Project were analysed to create GutFeelingKB.
  </span><br>
  <br>Version:
  <span property="version">4.0 </span>
  Citation:
  <span property="citation">PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/31509535/">31509535</a></span>
  <br>Downloads Link:
  <a href="https://hive.biochemistry.gwu.edu/gfkb">https://hive.biochemistry.gwu.edu/gfkb</a>

</li>
<br>

<li>
  <span property="name">Polyester Simulated RNA-seq Reads for Chromosome 22</span><br>
  <span property="abstract">
        Simulated RNA-seq reads were generated using the R package polyester for Chromosome 22 of the human reference genome GRCh38. Two samples were generated, with each sample containing a unique 2 transcripts that are expressed at 20 fold higher then normal to serve as positive controls i.e. Sample 1 has transcripts A and B expressed at 20 fold and sample 2 has transcripts C and D expressed at 20 fold. These reads can be used for testing RNA-seq analysis pipelines and to gauge any variability an analysis has on validating the 20 fold difference of the positive control transcripts between samples. 
  </span><br>
  <br>Version:
  <span property="version">1.0 </span>
 More info:
  <span property="Polyester information"><a href="https://bioconductor.org/packages/release/bioc/html/polyester.html">Bioconductor polyester page</a></span>
  
  <br>Downloads Link:
  <a href="HTMLROOT/content/soft_links/sample_01_1.fasta">sample_01_01.fasta</a>
  Summary:
  <span property="Summary">Forward reads for sample 1. </span>
  Size:
  <span property="size">1.7  GB</span>
  Format:
  <span property="encodingFormat">FASTA</span>
  <br>
  
  <br>Downloads Link:
  <a href="HTMLROOT/content/soft_links/sample_01_2.fasta">sample_01_2.fasta</a>
  Summary:
  <span property="Summary">Reverse reads for sample 1. </span>
  Size:
  <span property="size">1.7  GB</span>
  Format:
  <span property="encodingFormat">FASTA</span>
  <br>
  
  <br>Downloads Link:
  <a href="HTMLROOT/content/soft_links/sample_02_1.fasta">sample_02_01.fasta</a>
  Summary:
  <span property="Summary">Forward reads for sample 2. </span>
  Size:
  <span property="size">1.8  GB</span>
  Format:
  <span property="encodingFormat">FASTA</span>
  <br>
  
  <br>Downloads Link:
  <a href="HTMLROOT/content/soft_links/sample_02_2.fasta">sample_02_02.fasta</a>
  Summary:
  <span property="Summary">Reverse reads for sample 2. </span>
  Size:
  <span property="size">1.8  GB</span>
  Format:
  <span property="encodingFormat">FASTA</span>
  <br>

<br>Downloads Link:
  <a href="HTMLROOT/content/soft_links/sim_tx_info.txt">sim_tx_info.txt</a>
  Summary:
  <span property="Summary">Summary of fold changes per transcript. </span>
  Size:
  <span property="size">142  KB</span>
  Format:
  <span property="encodingFormat">TXT</span>
  <br>

</li>





</ol>
</td>
</tr>
<tr>
<td style="width:100%; vertical-align: top;" colspan="2">
<br>
</body>
